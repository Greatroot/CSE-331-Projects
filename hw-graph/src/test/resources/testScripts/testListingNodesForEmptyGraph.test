# Test to see if building an empty graph results in no nodes

CreateGraph emptyGraph
ListNodes emptyGraph
