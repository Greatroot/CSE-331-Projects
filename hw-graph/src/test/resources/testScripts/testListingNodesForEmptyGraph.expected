# Test to see if building an empty graph results in no nodes

created graph emptyGraph
emptyGraph contains: